module/rest/src/interceptor/cors.ts

```diff
@@ -5,12 +5,12 @@ import { Request, Response, RouteConfig, Method } from '../types';
 import { RestInterceptor } from './interceptor';
 
 @Config('rest.cors')
-class RestCorsConfig {
+export class RestCorsConfig {
```

Comment:
And we can definitely expose this, but out of curiosity, why would you want this exposed?

Reply:
we use the origins on cors for our authentication routes to redirect back to there since we have saml set up.